src/SFA.DAS.FindApprenticeshipTraining.Api/Controllers/TrainingCoursesController.cs

```diff
@@ -27,15 +28,16 @@ public class TrainingCoursesController : ControllerBase
         }
 
         [HttpGet]
-        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null)
+        public async Task<IActionResult> GetList( [FromQuery] string keyword = "", [FromQuery] List<Guid> routeIds = null, [FromQuery]List<int> levels = null, [FromQuery]int orderBy = 0)
```

Comment:
web is sending this as a string. semantic web and all.

Reply:
This is fixed